packages/task/src/browser/task-configurations.ts

```diff
@@ -60,7 +61,7 @@ export class TaskConfigurations implements Disposable {
     protected taskCustomizationMap = new Map<string, TaskCustomization[]>();
 
     /** last directory element under which we look for task config */
-    protected readonly TASKFILEPATH = '.theia';
+    protected configFolderName: string;
```

Comment:
please remove `configFolderName` and `getConfigFileUri`, that's left overs from previous refactorings, nobody is using them

Reply:
I am happy to remove unused protected methods. See here: https://github.com/eclipse-theia/theia/pull/7214/files#diff-65aa48cf1be5fc5d2c202882762b24c4R249

Reply:
I think we can do it here, we won't break anybody

Reply:
> I am happy to remove unused protected methods

> I think we can do it here,

With pleasure. 👍